sampleModels/boxAnimated/glTF-MaterialsCommon/boxAnimated.gltf

```diff
@@ -349,7 +360,7 @@
     },
     "nodes": {
         "Camera-camera002Node": {
-            "camera": "",
+            "camera": "camera_0",
```

Comment:
Did you run this through the converter to regenerate this?  If not, and this was added by hand, we could also just remove this node (`Camera-camera002Node`) since glTF does not require it.